x/ratelimit/keeper/msg_server_test.go

```diff
@@ -73,22 +113,18 @@ func (s *KeeperTestSuite) TestMsgServer_UpdateRateLimit() {
 
 	denom := updateRateLimitMsg.Denom
 	channelId := updateRateLimitMsg.ChannelId
-	channelValue := int64(100)
+	channelValue := sdk.NewInt(100)
 
-	// Mint tokens for generating channel value
-	err := s.App.BankKeeper.MintCoins(s.Ctx, minttypes.ModuleName, sdk.NewCoins(sdk.NewInt64Coin(updateRateLimitMsg.Denom, channelValue)))
-	s.Require().NoError(err)
+	// Create channel and
+	s.createChannel(channelId)
+	s.createChannelValue(denom, channelValue)
 
 	// Attempt to update a rate limit that does not exist
-	_, err = s.GetMsgServer().UpdateRateLimit(sdk.WrapSDKContext(s.Ctx), &updateRateLimitMsg)
+	_, err := s.GetMsgServer().UpdateRateLimit(sdk.WrapSDKContext(s.Ctx), &updateRateLimitMsg)
```

Comment:
What about removing GetMsgServer function and instead, adding MsgServer field in KeeperTestSuite struct?
like similar to what you did for QueryClient

Reply:
I'm glad you mentioned this! I thought the same thing when I added the query client but forgot to go back and try it

I had assumed we set it up with the Getter intentionally for stakeibc, but I can't seem to recall why we needed to do that.

Reply:
Ok